mods/authentication/src/utils/ITokenManager.ts

```diff
@@ -0,0 +1,7 @@
+import JwtPayload from './jwtPayload'
+export default interface ITokenManager {
+  readPublicKey(): Promise<string>
```

Comment:
Remove the `readPublicKey` and `readPrivateKey` from the interface. Also rename the interface to `itoken_manager.ts`